### PR TITLE
added code for constructing tree from inorder and preorder traversals…

### PR DESCRIPTION
**Pull request by pankajpoply45**
… in python

### CODE REVIEW

**Comment by pankajpoply45** [comment]
done with issue

File: Tree_From_Preorder_and_Inorder_Traversal_in_Python.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    for i in range(ilo,ihi+1):
+        if (ino[i] == pre[plo]):
+            si = i
+            break
+    nel = si - ilo
+    #constructing left subtree with help of recursion
+    root.left = constructPre(pre, plo + 1, plo + nel, ino, ilo, si - 1)
+    root.right = constructPre(pre, plo + nel + 1, phi, ino, si + 1, ihi)
+    return root
+#function for displaying the tree
```
Comment (jainaman224) [inline Tree_From_Preorder_and_Inorder_Traversal_in_Python.py]: Add new line above this and do this for function as well.

Comment (pankajpoply45) [reply Tree_From_Preorder_and_Inorder_Traversal_in_Python.py]: Sorry not understood what you mean to say @jainaman224 . Please explain again . I am new to open source that's why getting problem.

Comment (jainaman224) [reply Tree_From_Preorder_and_Inorder_Traversal_in_Python.py]: You should add new lines above every function.
So it is easier to understand the code.

**Comment by pankajpoply45** [comment]
@jainaman224  done with adding lines

File: Tree_From_Preorder_and_Inorder_Traversal_in_Python.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+#constructing a binary tree class
+
+class BinaryTree:
+    def __init__(self,preorder,ion) :
+        self.root = constructPre(preorder, 0, len(preorder) - 1, ino, 0, len(ino) - 1)
+
+#function for constructing the tree from given inorder and preorder traversal
+
+def constructPre(pre,plo,phi,ino,ilo,ihi):
```
Comment (jainaman224) [inline Tree_From_Preorder_and_Inorder_Traversal_in_Python.py]: Please use more understandable variable names and make two different program for inorder and preorder.

**Comment by pankajpoply45** [comment]
@jainaman224  done with changing names. Please review.

**Comment by jainaman224** [comment]
Please fix file name.

**Comment by pankajpoply45** [comment]
done with fixing file name. Please review it @jainaman224

**Pull request merged**